### PR TITLE
Jenkins: Fix v1a4 jobs, and update CI description

### PR DESCRIPTION
**Pull request by maelk**

### CODE REVIEW

File: jenkins/README.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ triggered on PR from metal3-dev-env, baremetal-operator and
 cluster-api-provider-metal3 repositories by commenting the commands below.
 The job result will be posted as a comment.
 
- * **/test-integration** run integration tests for V1alpha3 on Ubuntu
- * **/test-centos-integration** run integration tests for V1alpha3 on
+ * **/test-integration** run integration tests for V1alpha4 on Ubuntu
+ * **/test-centos-integration** run integration tests for V1alpha4 on
+   CentOS
+ * **/test-v1a3-integration** run integration tests for V1alpha3 on Ubuntu
+ * **/test-v1a3-centos-integration** run integration tests for V1alpha3 on
```
Comment (fmuyassarov) [inline jenkins/README.md:36]:
```suggestion
 * **/test-integration** run integration tests for V1alpha3 on Ubuntu
 * **/test-centos-integration** run integration tests for V1alpha3 on
   CentOS
 * **/test-v1a4integration** run integration tests for V1alpha4 on Ubuntu
 * **/test-v1a4-centos-integration** run integration tests for V1alpha4 on
```

Comment (fmuyassarov) [reply jenkins/README.md:36]: Sorry, I forgit about the changes in Nordix JJB.

Comment (maelk) [reply jenkins/README.md:36]: no, this is on purpose, I'm updating the job to use v1a4 by default

Comment (fmuyassarov) [reply jenkins/README.md:36]: I just didn't see your Nordix JJB PR before this.

**Pull request merged**